src/main/java/ch/njol/skript/sections/SecConditional.java

```diff
@@ -130,9 +131,19 @@ public boolean init(Expression<?>[] exprs,
 					Skript.error("'then' has to placed just after a multiline 'if' or 'else if' section");
 				}
 				return false;
-			} else if (!lastIf.multiline && type == ConditionalType.THEN) {
-				Skript.error("'then' has to placed just after a multiline 'if' or 'else if' section");
-				return false;
+			} else if (type == ConditionalType.THEN) {
+				/* if this is a 'then' section, the preceding conditional has to be a multiline conditional section
+				/ otherwise, you could put a 'then' section after a non-multiline 'if'. for example:
+				/  if 1 is 1:
+				/    set {_example} to true
+				/  then: # this shouldn't be possible
+				/    set {_uh oh} to true
+				*/
```

Comment:
```suggestion
				/* if this is a 'then' section, the preceding conditional has to be a multiline conditional section
				 * otherwise, you could put a 'then' section after a non-multiline 'if'. for example:
				 * if 1 is 1:
				 *   set {_example} to true
				 * then: # this shouldn't be possible
				 *   set {_uh oh} to true
				 */
```
i think asterisks are more appropriate?